framework/web/UrlManager.php

```diff
@@ -190,6 +185,14 @@ public function init()
     }
 
     /**
+     * return class name suffixed with 
+     **/
+    protected function getCacheKey() {
+        return __CLASS__.'.'.Yii::getAlias('@app');
```

Comment:
Check my other pr with different approach. There is no need for getters nor setters